File: DataSources/Databricks/server/spring-boot-jersey/src/main/java/com/server/reveal/AuthenticationProvider.java
package com.server.reveal;

import com.infragistics.reveal.sdk.api.IRVAuthenticationProvider;
import com.infragistics.reveal.sdk.api.IRVDataSourceCredential;
import com.infragistics.reveal.sdk.api.IRVUserContext;
import com.infragistics.reveal.sdk.api.RVBearerTokenDataSourceCredential;
import com.infragistics.reveal.sdk.api.model.RVDashboardDataSource;
import com.infragistics.reveal.sdk.api.model.RVDatabricksDataSource;

public class AuthenticationProvider implements IRVAuthenticationProvider {
	@Override
	public IRVDataSourceCredential resolveCredentials(IRVUserContext userContext, RVDashboardDataSource dataSource) {
        if (dataSource instanceof RVDatabricksDataSource) {
			return new RVBearerTokenDataSourceCredential("your_personal_access_token", "your_userid");
		} 
		return null;
	}
}

File: DataSources/Databricks/server/aspnet/RevealSdk.Server/Reveal/AuthenticationProvider.cs
using Reveal.Sdk;
using Reveal.Sdk.Data;
using Reveal.Sdk.Data.Databricks;

namespace RevealSdk.Server.Reveal
{
    public class AuthenticationProvider : IRVAuthenticationProvider
    {
        public Task<IRVDataSourceCredential> ResolveCredentialsAsync(IRVUserContext userContext,
            RVDashboardDataSource dataSource)
        {
            IRVDataSourceCredential userCredential = null;
            if (dataSource is RVDatabricksDataSource)
            {
                userCredential =
                    new RVPersonalAccessTokenDataSourceCredential("your_personal_access_token");
            }

            return Task.FromResult(userCredential);
        }
    }
}